test/unit/routes.test.js

```diff
@@ -44,9 +69,41 @@ describe('routes', () => {
 
       expect(replaceState.callCount).to.equal(0);
 
-      requireAuth(api)(null, replaceState);
+      requireAuth(api)(null, replaceState, () => {
+        expect(replaceState.callCount).to.equal(0);
+        done();
+      });
+
+    });
+
+    it('should update route to /terms with nextState path if user is authenticated and has not accepted the terms', (done) => {
+      let api = {
+        user : {
+          isAuthenticated: sinon.stub().returns(true),
+          get: (cb) => {
+            cb(
+              null,
+              {
+                userid: 'a1b2c3',
+                profile: {
+                  patient: {}
+                },
+                termsAccepted: false
```

Comment:
Before terms are accepted the field comes back as `''` (empty string) - should be updated in these test fixtures ideally.